src/PrestaShopBundle/Form/Admin/Sell/Order/Invoices/InvoiceOptionsDataProvider.php

```diff
@@ -82,21 +84,63 @@ public function setData(array $data)
      *
      * @param array $data
      *
-     * @return array Array of errors if any
+     * @return void
+     *
+     * @throws TypeException|DataProviderException
      */
-    private function validate(array $data)
+    private function validate(array $data): void
```

Comment:
Could you add unit test on this class, please ?

Reply:
Added the tests